_data/languages.json

```diff
@@ -6707,272 +6714,317 @@
     },
     {
         "codes": [
-            "huyu"
+            "ch"
```

Comment:
By the way, we could easily add the codes for every language - we have the mapping from 3-letter codes to 2-letter codes in api_languages.json, could reverse it to add all these.